chain-vote/src/private_voting/mod.rs

```diff
@@ -0,0 +1,14 @@
+//! Implementation of the Unit Vector ZK argument presented by
+//! Zhang, Oliynykov and Balogum in
+//! ["A Treasury System for Cryptocurrencies: Enabling Better Collaborative Intelligence"](https://www.ndss-symposium.org/wp-content/uploads/2019/02/ndss2019_02A-2_Zhang_paper.pdf).
+//! We use the notation presented in the technical
+//! [spec](https://github.com/input-output-hk/treasury-crypto/blob/master/docs/voting_protocol_spec/Treasury_voting_protocol_spec.pdf),
```

Comment:
Unfortunately, this link gets me to 404.

Reply:
This may be due to limited permissions.

Reply:
ah, yes, I thought this was public. Do you think it makes sense to note that it needs permissions (giving that we don't expect this to be a public library)? Or rather modify the docs and notation to fit with the published paper?

Reply:
I'll talk with the research team then 👍